runhouse/main.py

```diff
@@ -719,12 +724,28 @@ def _start_server(
         raise typer.Exit(1)
 
 
-@app.command()
-def start(
+###############################
+# Server CLI commands
+###############################
+
+
+@server_app.command("start")
+def server_start(
```

Comment:
wondering if we should handle more gracefully trying to start the server if it's already running - and instead direct the user to run `restart` - for example when running locally: 

```
INFO:__main__:Launching Runhouse API server with den_auth=False and on host=0.0.0.0 and use_https=False and port_arg=32300
INFO:     Started server process [41325]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
ERROR:    [Errno 48] error while attempting to bind on address ('0.0.0.0', 32300): address already in use
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
```